internal/util/search.go

```diff
@@ -41,3 +41,39 @@ func IsUnderTiUP(binpath string) bool {
 
 	return false
 }
+
+func GetDisplayRole(builtinRole string, customRoles []string) string {
+	if builtinRole != "" {
+		customRoles = append(customRoles, TrimRolePrefix(builtinRole))
+	}
+	displayRoles := make([]string, 0, len(customRoles))
+
+	for _, role := range customRoles {
+		var displayRole string
+		switch role {
+		case ADMIN_ROLE, ADMIN:
+			displayRole = ADMIN_DISPLAY
+		case READWRITE_ROLE, READWRITE:
+			displayRole = READWRITE_DISPLAY
+		case READONLY_ROLE, READONLY:
+			displayRole = READONLY_DISPLAY
+		default:
+			displayRole = role
+		}
+
```

Comment:
I'm a bit lost why we need to apply this logic to every role? Isn't this specific to builtinrole?

Reply:
This is a mistake, the judgement should only apply to builtinRole